charts/gardener-extension-shoot-cert-service/templates/_helpers.tpl

```diff
@@ -8,6 +8,7 @@ apiVersion: shoot-cert-service.extensions.config.gardener.cloud/v1alpha1
 kind: Configuration
 issuerName: {{ required ".Values.certificateConfig.defaultIssuer.name is required" .Values.certificateConfig.defaultIssuer.name }}
 restrictIssuer: {{ required ".Values.certificateConfig.defaultIssuer.restricted is required" .Values.certificateConfig.defaultIssuer.restricted }}
+defaultRequestsPerDayQuota: {{ required ".Values.certificateConfig.defaultRequestsPerDayQuota is required" .Values.certificateConfig.defaultRequestsPerDayQuota }}
```

Comment:
For backwards compatibility can you make this optional?

Reply:
done